packages/api/tests/task.test.js

```diff
@@ -2013,7 +2010,14 @@ describe('Task tests', () => {
       patchTaskDateRequest({ user_id, farm_id }, patchTaskDateBody, task_id, async (err, res) => {
         expect(res.status).toBe(200);
         const updated_task = await getTask(task_id);
-        expect(updated_task.due_date.toISOString().split('T')[0]).toBe(due_date);
+        const dueDateFromDb = new Date(
```

Comment:
@MwayaB Please see if this causes the test to pass for you. The change is based on this attempt to manually simulate the problem.
![image](https://user-images.githubusercontent.com/21227171/159338018-008b55be-d14b-4675-9ad0-c084a39324a8.png)